<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddParameter</class>
 <widget class="QWidget" name="AddParameter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,0">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="name_l">
             <property name="text">
              <string>Nazwa</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="name"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="symbol_l">
             <property name="text">
              <string>Symbol</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="symbol"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="parameterType_l">
             <property name="text">
              <string>Typ</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="parameterType"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="value_hl">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <item>
            <widget class="QCheckBox" name="hasLimitedValue">
             <property name="text">
              <string>Ograniczenie zakresu</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="minVal_l">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Min</string>
             </property>
             <property name="buddy">
              <cstring>minVal</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QEngineerSpinBox" name="minVal">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="minimum">
              <double>-999999999.000000000000000</double>
             </property>
             <property name="maximum">
              <double>999999999.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="maxVal_l">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Max</string>
             </property>
             <property name="buddy">
              <cstring>maxVal</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QEngineerSpinBox" name="maxVal">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="minimum">
              <double>-999999999.000000000000000</double>
             </property>
             <property name="maximum">
              <double>999999999.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="length_hl">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <item>
            <widget class="QCheckBox" name="hasLimitedLength">
             <property name="text">
              <string>Ograniczenie długości</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="minLength_l">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Min</string>
             </property>
             <property name="buddy">
              <cstring>minLength</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="minLength">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="maxLength_l">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Max</string>
             </property>
             <property name="buddy">
              <cstring>maxLength</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="maxLength">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="default_hl">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <item>
            <widget class="QCheckBox" name="hasDefault">
             <property name="text">
              <string>Wartość domyślna</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="default_l">
             <property name="text">
              <string/>
             </property>
             <property name="buddy">
              <cstring>defaultValue</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="defaultValue">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="description_l">
         <property name="text">
          <string>Opis</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="buddy">
          <cstring>description</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="description">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>10</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addParameter">
         <property name="text">
          <string>Dodaj parametr</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="status">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>All parameters</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="parametersView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QEngineerSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qengineerspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
